assets/compat/config_spec.yaml

```diff
@@ -419,6 +455,18 @@ advanced:
         using a Child Pays For Parent transaction.
       nullable: false
       default: false
+    protocol-disable-script-enforced-lease:
+      type: boolean
+      name: Disable Script Enforced Channel Leases
+      description: >-
+        Set to disable support for script enforced lease channel commitments. If not
+        set, lnd will accept these channels by default if the remote channel party
+        proposes them. Note that lnd will require 1 UTXO to be reserved for this
+        channel type if it is enabled.
+
+        Note: This may cause you to be unable to close a channel and your wallets may not understand why
+      nullable: false
+      default: false
```

Comment:
In which situations is this needed?

Reply:
This system is used for channel leases that are created through things like Pool.